### PR TITLE
FTDI fixes in master

### PR DESCRIPTION
**Pull request by vinodkoul**
Noticed that the master is broken so the ftdi patch fixes that. Also noticed warning in ftdi for sleep so fixed that as well

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,7 +6,7 @@ SERVER := cdba-server
 all: $(CLIENT) $(SERVER)
 
 CFLAGS := -Wall -g -O2
-LDFLAGS := -ludev -lyaml -lftdi -lusb
+LDFLAGS := -ludev -lyaml -lftdi1 -lusb
```
Comment (calebccff) [inline Makefile:9]: Both libftdi and libftdi1 can be supported with the following:

```diff
diff --git a/Makefile b/Makefile
index 2c8facdb7887..db44b455c7cc 100644
--- a/Makefile
+++ b/Makefile
@@ -5,8 +5,8 @@ SERVER := cdba-server

 all: $(CLIENT) $(SERVER)

-CFLAGS := -Wall -g -O2
-LDFLAGS := -ludev -lyaml -lftdi -lusb
+CFLAGS := -Wall -g -O2 $(shell pkg-config --silence-errors --cflags libfdti) $(shell pkg-config --silence-errors --cflags libftdi1)
+LDFLAGS := -ludev -lyaml -lusb $(shell pkg-config --silence-errors --libs libftdi) $(shell pkg-config --silence-errors --libs libftdi1)

 CLIENT_SRCS := cdba.c circ_buf.c
 CLIENT_OBJS := $(CLIENT_SRCS:.c=.o)
```

File: ftdi-gpio.c [hunk 2]
```diff
@@ -38,11 +38,12 @@
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
+#include <unistd.h>
 
 #include "cdba-server.h"
 #include "ftdi-gpio.h"
 
-#include "ftdi.h"
+#include <libftdi1/ftdi.h>
```
Comment (calebccff) [inline ftdi-gpio.c:46]: I think adding the include directory to CFLAGS is preferable to having the code include `libftddi1/ftdi.h`.

**Comment by superna9999** [comment]
With @calebccff additions, it would be a great fix !

File: README [hunk 3]
```diff
@@ -3,8 +3,8 @@ The CDBA control tool is used for remotely booting images on development boards
 attached using a CDB Assist [https://github.com/sonyxperiadev/CDB-Assist] or Conmux.
 
 = Dependencies
-sudo apt-get install libudev-dev libyaml-dev for debian systems
-dnf install systemd-devel libyaml-devel for fedora systems
+sudo apt-get install libudev-dev libyaml-dev libftdi libftdi-dev for debian systems
```
Comment (lumag) [inline README:6]: Please don't install library packages manually. Installing `libftdi-dev` should be enough.

**Comment by konradybcio** [comment]
please resubmit just 03e99e66a90119577108fd1d945bfddf031e5f03

**Pull request closed without merging**